global.R

```diff
@@ -88,7 +89,7 @@ default_budget_prices_html <- switch(utility_code,
 # Read data from file and rename the columns to be compatable with internal calls
 df <- read_data(test_file, cust_col="cust_loc_id", usage_col="usage_ccf", month_col="usage_month", 
                 year_col="usage_year", et_col="usage_et_amount", hhsize_col="cust_loc_hhsize", 
-                irr_area_col="cust_loc_irr_area_sf", rate_code_col= "cust_loc_class", 
+                irr_area_col="cust_loc_irr_area_sf", rate_code_col= "cust_loc_class", cust_class_col = "cust_loc_class_from_utility",
```

Comment:
@christophertull sounds good on me opening a pull request after I incorporate your suggested changes..i have no idea with the workflow of github either :-)